word_language_model/README.md

```diff
@@ -21,24 +27,28 @@ The `main.py` script accepts the following arguments:
 
 ```bash
 optional arguments:
-  -h, --help         show this help message and exit
-  --data DATA        location of the data corpus
-  --model MODEL      type of recurrent net (RNN_TANH, RNN_RELU, LSTM, GRU)
-  --emsize EMSIZE    size of word embeddings
-  --nhid NHID        number of hidden units per layer
-  --nlayers NLAYERS  number of layers
-  --lr LR            initial learning rate
-  --clip CLIP        gradient clipping
-  --epochs EPOCHS    upper epoch limit
-  --batch_size N     batch size
-  --bptt BPTT        sequence length
-  --dropout DROPOUT  dropout applied to layers (0 = no dropout)
-  --decay DECAY      learning rate decay per epoch
-  --tied             tie the word embedding and softmax weights
-  --seed SEED        random seed
-  --cuda             use CUDA
-  --log-interval N   report interval
-  --save SAVE        path to save the final model
+  -h, --help                       show this help message and exit
+  --data DATA                      location of the data corpus
+  --model MODEL                    type of recurrent net (RNN_TANH, RNN_RELU, LSTM, GRU)
+  --emsize EMSIZE                  size of word embeddings
+  --nhid NHID                      number of hidden units per layer
+  --nlayers NLAYERS                number of layers
+  --lr LR                          initial learning rate
+  --clip CLIP                      gradient clipping
+  --epochs EPOCHS                  upper epoch limit
+  --batch_size N                   batch size
+  --bptt BPTT                      sequence length
+  --dropout DROPOUT                dropout applied to layers (0 = no dropout)
+  --decay DECAY                    learning rate decay per epoch
+  --tied                           tie the word embedding and softmax weights
+  --seed SEED                      random seed
+  --cuda                           use CUDA
+  --log-interval N                 report interval
+  --save SAVE                      path to save the final model
+  --transformer_head N             the number of heads in the encoder/decoder of the transformer model
```

Comment:
Can some of this be meaningfully folded into the other arguments? Or put differently, can we recycle some of the existing arguments for the Transformer model? For example nlayers. Do we need separate flags for the encoder and decoder?